### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by JakobGM**
First draft of contributing guidelines for new contributors.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+Here we describe how you can help out, either by improving the documentation, submitting issues, or commit pull requests.
+
+If you end up contributing, please consider adding yourself to the file ``CONTRIBUTORS.rst``.
+
+.. _contributing_issues:
+
+Bug reports and feature requests
+================================
+
+Bug reports and feature requests can be submitted in the form of a `GitHub issue <https://github.com/JakobGM/astrality/issues/new>`_.
```
Comment (sshashank124) [inline CONTRIBUTING.rst]: Maybe just link to /issues and mention to first check existing open/closed issues to see if a similar topic has been discussed before

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should create a separate python3.6 "virtual environment" exclusively for Astrality.
+If this is new to you, take a look at the `official tutorial <https://docs.python.org/3/tutorial/venv.html>`_ for ``venv``.
+A quick summation:
+
+.. code-block:: console
+
+    python3.6 -m venv astrality-env
+    source astrality-env/bin/activate
```
Comment (sshashank124) [inline CONTRIBUTING.rst]: Maybe mention that this is for switching into the virtual environment. General flow is to switch into the virtual environment using the source activate command, make/test changes, deactivate to leave the virtual environment. Most shells will have an updated prompt showing the name of the current virtual environment you are working in. You can also run `echo $VIRTUAL_ENV` to see the name (if any) of the virtual environment you are currently in

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,234 @@
+=================
+How to contribute
+=================
+
+First, thanks for considering contributing to Astrality, that means a lot!
+Here we describe how you can help out, either by improving the documentation, submitting issues, or commit pull requests.
```
[the remark this answers is not in the document]
Comment (JakobGM) [reply CONTRIBUTING.rst]: Fixed!

File: CONTRIBUTING.rst (hunk 1 above)
Comment (JakobGM) [reply CONTRIBUTING.rst]: Done!

File: CONTRIBUTING.rst (hunk 2 above)
Comment (JakobGM) [reply CONTRIBUTING.rst]: I don't know to what degree we should document an external tool like ``venv``, as there are hundreds of ways to interact with it, ``pyenv``, ``venv``, ``virtualenvwrapper``, and so on...

Well, I added a short description of the workflow anyway, but if we ever get anyone else to contribute, I think its a relatively safe assumption to say that they already know how to use python virtual environments.

**Comment by JakobGM** [comment]
OK, I've rewritten the guideline according to you feedback. Let me know if there is anything you would like different in the new sections, and let's merge it afterwards ;)

**Comment by JakobGM** [comment]
I've also included a section explaining how to debug Astrality by setting a more granular logging level.

**Comment by JakobGM** [comment]
Great, thanks for the feedback, I'll merge it in!

It will be linked from readthedocs too, where it will be rendered in the same style as the rest of the documentation.

**Pull request merged**